build.sc

```diff
@@ -116,7 +121,7 @@ trait OsLibTestModule extends ScalaModule with TestModule{
     millSourcePath / s"src-$platformSegment"
   )
 
-  def testFrameworks = Seq("utest.runner.Framework")
+  def testFrameworks = Seq("utest.runner.Framework", "org.scalacheck.ScalaCheckFramework")
```

Comment:
We just deprecated muitiple test frameworks per test module in mill. Either call ScalaCheck API from utests or create a seconds test module using the other test framework.

Reply:
I will do some digging and figure out how to do that

Reply:
https://github.com/com-lihaoyi/os-lib/pull/70/commits/99ae578fb2730b0c81afa1ed61dc3d623994db92